### PR TITLE
4.0.x

### PR DESCRIPTION
**Pull request by wwelling**
Pending approvals.

### CODE REVIEW

File: src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="input-group input-text-with-definition">
```
Comment (jcreel) [inline src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html]: Where is this input being used?

File: src/main/webapp/app/views/admin/view.html [hunk 2]
```diff
@@ -8,15 +8,13 @@
...
-        <span class="glyphicon glyphicon-refresh spinning"></span>
-        Loading Submission...</span>
+      <span ng-if="!submission"><span class="glyphicon glyphicon-refresh spinning"></span>Loading Submission...</span>
       <span ng-if="submission">{{title}}</span>
     </h1>
     <hr />
 
     <vireo-tabs ng-show="loaded">
-      <vireo-tab class="item-view" ng-repeat="workflowStep in submission.submissionWorkflowSteps" ng-if="showTab(workflowStep)" label="{{workflowStep.name}}" path="admin/view/{{submission.id}}/{{workflowStep.id}}" view="views/admin/info/workflowStep.html"></vireo-tab>
+      <vireo-tab class="item-view" reload="false" ng-repeat="workflowStep in submission.submissionWorkflowSteps" ng-if="showTab(workflowStep)" label="{{workflowStep.name}}" path="admin/view/{{submission.id}}/{{workflowStep.id}}" view="views/admin/info/workflowStep.html"></vireo-tab>
```
Comment (jcreel) [inline src/main/webapp/app/views/admin/view.html]: remove extraneous attribute

File: src/main/webapp/app/views/directives/fieldProfile.html [hunk 3]
```diff
@@ -1,23 +1,27 @@
 <form id="{{profile.fieldPredicate.value}}" class="field-profile input-text row" name="fieldProfileForm" novalidate autocomplete="off">
   <div class="form-group">
     <div class="col-xs-12 col-sm-2">
-      <label ng-class="{'required': !profile.optional}">{{profile.fieldGlosses[0].value}}:</label>
+      <label>
+        <strong>
```
Comment (jcreel) [inline src/main/webapp/app/views/directives/fieldProfile.html]: See about making just the required strong.

File: src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html (hunk 1 above)
Comment (wwelling) [reply src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html]: It is not. 

Here is the InputType: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/resources/input_types/SYSTEM_Input_Types.json#L27

Here is the admin template: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html

Here is the student template: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/webapp/app/views/inputtype/input-text_with_definition.html

It is not used in the initial field profiles anywhere.

https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/resources/organization/SYSTEM_Organization_Definition.json

Simply removing from default Input_Tyes.json and deleting the two templates is all that is needed to remove this relic.

**Comment by wwelling** [comment]
Do we want to remove the input with definition?

**Pull request merged**